E1-ALRF/ALLP/README.md

```diff
@@ -0,0 +1,167 @@
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+:construction: under construction :construction:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+    - *IfcAlignment*
+    - *IfcAlignmnetHorizontal*
+    - *IfcAlignmentVertical*
+    - *IfcAlignmentSegment*
+    - *IfcAlignmentHorizontalSegment*
+    - *IfcAlignmentVerticalSegment*
+    - *IfcAxis2PlacementLinear*
+    - *IfcAxis2Placement2D*
+    - *IfcAxis2Placement3D*
+    - *IfcCircle*
+    - *IfcClothoid*
+    - *IfcCompositeCurve*
+    - *IfcCurveSegment*
+    - *IfcLine*
+    - *IfcLinearPlacement*
+    - *IfcLocalPlacement*
+    - *IfcGeometricRepresentationContext*
+    - *IfcGeometricRepresentationSubContext*
+    - *IfcGradientCurve*
+    - *IfcMapConversion*
+    - *IfcPolyline*
+    - *IfcProject*
+    - *IfcProjectedCRS*
+    - *IfcProductDefinitionShape*
+    - *IfcReferent*
+    - *IfcRelPositions*
+    - *IfcSite*
+    - *IfcShapeRepresentation*
+
+</details>
+
+<details><summary>Concept Templates</summary>
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+    - *Alignment Layout*
+    - *Alignment Geometry*
+    - *Alignment Geometry Gradient*
+    - *Project Global Positioning*
+    - *Spatial Containment*
+
+</details>
+
+## Test Case Imports
+Test instructions are defined with a modular approach to reduce repetition of validation criteria and test content, and improve vendors ability to solve issues and bugs. therefore this test instruction *imports/reuses* the following Test instructions and entities with the relevant associated validation criteria.
+
+:construction: under construction :construction:
+
+<details><summary>Imports & Reuses</summary>
+
+| TI Code                                  | Test Instruction Title    | Comments                     |
+|------------------------------------------|---------------------------|------------------------------|
+| [IFC4.3AbRV_E0_MSTP](../../E0-SCFD/MSTP) | Model Setup & Positioning | PROJ-01 imported along with RCTX-01 and associated configuration and history data |
+| [IFC4.3AbRV_E1_ALRF](../../E1-ALRF/ALIG) | Alignment Curves | Valid alignment geometry |
+
+</details>
+
+## Usages, Constraints & Logic
+The following itemised restrictions and constraints shall be placed on IFC Entities & Concept Templates:
```

Comment:
What to write here and what to write under "Validation Criteria" needs to be clarified => Discussion!